### PR TITLE
Add FXIOS-6128 [v114] Test updates on CCInputField

### PR DESCRIPTION
**Pull request by nishant2718**
# [FXIOS-6128](https://mozilla-hub.atlassian.net/browse/FXIOS-6128)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/TODO)

### Description
Updating tests after changes in CreditCardInputField

### Pull requests checks where applicable
- [x] Fill in the three TODOs above (tickets number and description of your work)
- [x] The PR name follows our [naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Accessibility implemented and tested (minimum Dynamic Text and VoiceOver)
- [x] Unit tests written and passing
- [ ] Documentation / comments for complex code and public methods

### CODE REVIEW

File: Tests/ClientTests/CreditCard/CreditCardInputFieldTests.swift [hunk 1]
```diff
@@ -4,13 +4,11 @@
...
 
-/*
 import Foundation
 import XCTest
 import SwiftUI
 @testable import Client
 
-
 class CreditCardInputFieldTests: XCTestCase {
     @State var testableString: String = ""
```
Comment (nbhasin2) [inline Tests/ClientTests/CreditCard/CreditCardInputFieldTests.swift:13]: Do you need the state var still? 🤔 
Is it possible to test without using state variables

Comment (nishant2718) [reply Tests/ClientTests/CreditCard/CreditCardInputFieldTests.swift]: Updated - no longer using this!

File: Tests/ClientTests/CreditCard/CreditCardInputFieldHelperTests.swift [hunk 2]
```diff
@@ -0,0 +1,78 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import XCTest
+
+@testable import Client
+
+class CreditCardInputFieldHelperTests: XCTestCase {
```
Comment (yoanarios) [inline Tests/ClientTests/CreditCard/CreditCardInputFieldHelperTests.swift:10]: The coverage is high in this class but there are some guards and if conditions (edge cases) which are perfect for unit testing

File: Tests/ClientTests/CreditCard/CreditCardInputFieldTests.swift [hunk 3]
```diff
@@ -4,13 +4,11 @@
...
 // Disabled: It will be updated in FXIOS-6128
 
-/*
 import Foundation
 import XCTest
 import SwiftUI
 @testable import Client
 
-
 class CreditCardInputFieldTests: XCTestCase {
```
Comment (yoanarios) [inline Tests/ClientTests/CreditCard/CreditCardInputFieldTests.swift:12]: Same comment as above great coverage but it will be good to add test for edge cases

**Review by yoanarios — COMMENTED** [review]
Great work in the amount and quality of test but I think there is still some cases where more test can be added like `CreditCardInputViewModel`

**Review by nbhasin2 — APPROVED** [review]

**Comment by nishant2718** [comment]
BR failed on a fixed in `main` issue, 
image

merging.

**Comment by nishant2718** [comment]
@yoanarios I apologize for the premature merge. We can sync offline to discuss areas that are still of concern and address it soon after.
